Configuration/StandardSequences/python/Eras.py

```diff
@@ -10,4 +10,25 @@ def __init__(self):
         self.bunchspacing25ns = cms.Modifier()
         self.bunchspacing50ns = cms.Modifier()
 
+        # These eras should not be set directly by the user.
+        self.run2_common = cms.Modifier()
+        self.run2_25ns_specific = cms.Modifier()
+        self.run2_50ns_specific = cms.Modifier()
+        self.run2_HI_specific = cms.Modifier()
+        self.stage1L1Trigger = cms.Modifier()
+        
+        # These are the eras that the user should specify
+        self.Run2_25ns = cms.ModifierChain( self.run2_common, self.run2_25ns_specific, self.stage1L1Trigger )
+        self.Run2_50ns = cms.ModifierChain( self.run2_common, self.run2_50ns_specific )
+        self.Run2_HI = cms.ModifierChain( self.run2_common, self.run2_HI_specific, self.stage1L1Trigger )
```

Comment:
Hi Mark,
as far as I know, the L1 Stage 1 trigger configuration for Heavy Ions is slightly different than that for pp/25ns.
How is that handled here ?